grub2_theme_preview/cli.py

```diff
@@ -234,6 +234,12 @@ def _grub2_directory(platform):
     return  '/usr/lib/grub/%s' % platform
 
 
+def _grub2_platform():
+    _BOOT_TYPE = 'efi' if os.path.exists('/sys/firmware/efi') else 'pc'
+    _PROC_TYPE = 'i386' if _BOOT_TYPE == 'pc' else platform.machine()
```

Comment:
Let's go with their naming `cpu` and `platform`, use one if rather than two for readability and embed that link. How about:
```
def _grub2_platform():
    if os.path.exists('/sys/firmware/efi'):
        cpu = platform.machine()
        platform = 'efi'
    else:
        # i.e. BIOS-based machines
        # https://www.gnu.org/software/grub/manual/grub/grub.html#Installation
        cpu = 'i386'
        platform = 'pc'

    return '%s-%s' % (cpu, platform)
```
?

Reply:
Awesome, will do.